scripts/update_and_rebuild_libmesh.sh

```diff
@@ -141,6 +141,15 @@ if [ -z "$go_fast" ]; then
     export INSTALL_BINARY="${SCRIPT_DIR}/../libmesh/build-aux/install-sh -C"
   fi
 
+  # This is a temprorary fix, see #15120
+  if [[ -n "$CPPFLAGS" ]]; then
+    export CPPFLAGS=${CPPFLAGS//-DNDEBUG/}
+    export CPPFLAGS=${CPPFLAGS//-O2/}
+  fi
+  if [[ -n "$CXXFLAGS" ]]; then
+    export CXXFLAGS=${CXXFLAGS//-O2/}
+  fi
+  
```

Comment:
Can't speak to the other changes, but this looks good.

Reply:
Other changes are from things already in next that I wanted to test with this.